test/test_trixi.jl

```diff
@@ -16,17 +16,38 @@ as absolute/relative tolerance.
 """
 macro test_trixi_include(elixir, args...)
 
-  local l2   = get_kwarg(args, :l2, nothing)
-  local linf = get_kwarg(args, :linf, nothing)
-  local atol = get_kwarg(args, :atol, 500*eps())
-  local rtol = get_kwarg(args, :rtol, sqrt(eps()))
+  local l2                   = get_kwarg(args, :l2, nothing)
+  local linf                 = get_kwarg(args, :linf, nothing)
+  local atol                 = get_kwarg(args, :atol, 500*eps())
+  local rtol                 = get_kwarg(args, :rtol, sqrt(eps()))
+  local run_without_coverage = get_kwarg(args, :run_without_coverage, true)
+  local run_with_coverage    = get_kwarg(args, :run_with_coverage, true)
```

Comment:
Any better name ideas for these guys?

Reply:
Why are there two arguments? This seems like a recipe for confusion in case they are both false. Also, having two keywords increases the chance that someone makes a silent typo that is never found, e.g., when using `run_without_coverge=false`.

Reply:
Aren't these kind of tests that we want to drop easier to discard at a higher level? For example, we could check whether coverage is turned on and avoid including the threaded tests if so. Similarly, we will probably need special cases for AD tests etc. Thus, I think we should probably remove these keyword arguments again and extend this PR or merge this PR (if @sloede approves) and make another PR to improve the handling of expensive `misc_partx` tests.

Reply:
I agree. Merging the first part that is already looking OK and then addressing other issues in a second PR seems like a good approach!

Reply:
I tried to avoid having some magic values (`:both`, `coverage`, ...) and to use binary switches instead. Anyway, I'm removing it again.